Adyen.xcodeproj/project.pbxproj

```diff
@@ -4754,6 +4757,7 @@
 				F917613825A30B5700D653BE /* ThreeDS2ComponentFingerprint.swift in Sources */,
 				F970143925D139BE007E74D9 /* VoucherComponentStyle.swift in Sources */,
 				F917613925A30B5700D653BE /* ThreeDS2ComponentFingerprintToken.swift in Sources */,
+				5A2D1951267368580082BCE9 /* AdyenActionComponentStyle.swift in Sources */,
```

Comment:
can this be just `ActionComponentStyle`, adyen is sort of redundant.

Reply:
👍